conf/st2.conf

```diff
@@ -34,6 +34,7 @@ base_path = /opt/stackstorm
 host = localhost
 port = 514
 facility = local7
+protocol = udp
```

Comment:
There should be a corresponding change in https://github.com/StackStorm/st2/blob/master/st2common/st2common/config.py#L90